api_app/script_analyzers/observable_analyzers/ipinfo.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+from api_app.exceptions import AnalyzerRunException, AnalyzerConfigurationException
+from api_app.script_analyzers import classes
+from intel_owl import secrets
+
+
+class IPInfo(classes.ObservableAnalyzer):
+    base_url: str = "https://ipinfo.io/"
+
+    def set_config(self, additional_config_params):
+        self.api_key_name = additional_config_params.get("api_key_name", "IPINFO_KEY")
+        self.__api_key = secrets.get_secret(self.api_key_name)
+
+    def run(self):
+        if not self.__api_key:
+            raise AnalyzerConfigurationException(
+                f"No API key retrieved with name: {self.api_key_name}."
+            )
+
+        params = {"token": self.__api_key}
+        uri = f"{self.observable_name}"
```

Comment:
this line is not necessary

Reply:
Is it regarding that empty line? or Should I pass them directly without assigning them to variables?

Reply:
second one I would say

Reply:
![Screenshot at 2020-11-10 17-19-54](https://user-images.githubusercontent.com/52322531/98670694-f763d880-234a-11eb-882a-150b7eced5c6.png)

This output is for `8.8.8.8`.

Reply:
Instead of amending the commit message, Shall I squash the commit? It will give an oppurtunity to change the commit message, Right?

Reply:
Yes, you could have done this before while squashing the commits